src/basics.md

```diff
@@ -1242,6 +1243,52 @@ fn main() {
 }
 ```
 
+[ex-datetime-from-input]: #ex-parse-datetime
+<a name="ex-parse-datetime"></a>
+## Parse string into DateTime struct
+[![chrono-badge]][chrono] [![cat-date-and-time-badge]][cat-date-and-time]
+
+Parses a [`DateTime`] struct from strings representing the well-known formats
+[RFC 2822], [RFC 3339], and a custom format, using
+[`DateTime::parse_from_rfc2822`], [`DateTime::parse_from_rfc3339`], and
+[`DateTime::parse_from_str`] respectively.
+
+Escape sequences that are available for the [`DateTime::parse_from_str`] can be
+found at [`chrono::format::strftime`]. Note that the [`DateTime::parse_from_str`]
+requires that such a DateTime struct must be creatable that it uniquely
+identifies a date and a time. For parsing dates and times without timezones, see
+[`NaiveDate::parse_from_str`], [`NaiveTime::parse_from_str`], and
```

Comment:
I would suggest to also use these functions in the actual code and just mention here about the types `NaiveDate` 
 / `Naivetime`  / `NaiveDateTime`

Reply:
> Is this what we are going after?

Pretty much. :+1:  Just the last nit listed above.